frontend/fingertips-frontend/components/molecules/result/searchResult.test.tsx

```diff
@@ -84,6 +84,35 @@ it('should contain expected text', () => {
   );
 });
 
+it('should display tag if indicator date is within one month of server date', () => {
+  const currentDate = new Date('December 7, 2024');
+  render(
+    <SearchResult
+      result={MOCK_DATA[0]}
+      searchState={initialSearchState}
+      handleClick={mockHandleClick}
+      serverDate={currentDate}
+    />
+  );
+  expect(screen.getAllByRole('strong').at(0)?.textContent).toContain(
+    'Updated in last month'
+  );
```

Comment:
I think making this assertion similar to how you have done the one below might be better

```
expect(screen.getByText('Updated in last month')).toBeInTheDocument();
```